cloudy_mozdef/packer/packer.json

```diff
@@ -30,7 +33,7 @@
         "sudo systemctl enable docker",
         "sudo mkdir -p /opt/mozdef/",
         "sudo git clone https://github.com/mozilla/MozDef /opt/mozdef",
-        "cd /opt/mozdef && sudo git checkout guardduty"
+        "cd /opt/mozdef && sudo git checkout reinvent"
```

Comment:
Do we want this PR in master or should this go in a `reinvent` branch?

Reply:
We should switch this back to master.  I missed this.  Need to come up with a better build / deploy for launching a stack off a branch based build.

Reply:
@andrewkrug Should I change this from `reinvent` to `guardduty` or from `reinvent` to `master`?